seastar/src/config_and_start_seastar.cc

```diff
@@ -54,4 +54,4 @@ int32_t run_int(app_template& app, int32_t argc, char** argv, IntFuture fut) {
 }
 
 } // namespace config_and_start_seastar
-} // namespace seastar
\ No newline at end of file
+} // namespace seastar
```

Comment:
This change is unrelated to this commit.
I don't have a problem with unrelated stylistic changed in a PR, but it should be in it's own commit

Reply:
Right, I moved this to a separate commit.